src/oemof/solph/flows/_invest_non_convex_flow_block.py

```diff
@@ -373,8 +372,8 @@ def _max_invest_rule(block, i, o):
             self.INVEST_NON_CONVEX_FLOWS, rule=_max_invest_rule
         )
 
-        self.min = nc.minimum_flow_constraint(self)
-        self.max = nc.maximum_flow_constraint(self)
+        self.min = self._minimum_flow_constraint()
+        self.max = self._maximum_flow_constraint()
 
         # z = x * y, where x is a binary variable (in our case `status`),
         # y is a continuous variable (in our case `invest`), and z denotes
```

Comment:
I like the idea of renaming the new parameter to `status_nominal`. It is more descriptive! But the old name stil exists in some parts. Here, it should be:
```
... and z denotes the new parameter `status_nominal`.
```

Similarly, for the following functions,
1. `_linearization_rule_invest_non_convex_one`
2. `_linearization_rule_invest_non_convex_two`
3. `_linearization_rule_invest_non_convex_three` 

the docstrings can be updated as follows:
```
Rule definition for the linearization of the `status_nominal`.
```

Reply:
I refactored the according code and changed the documentation. Do you like it?

https://oemof-solph.readthedocs.io/en/feature-restucture_flow/reference/oemof.solph.flow.html#oemof.solph.flows._invest_non_convex_flow_block.InvestNonConvexFlowBlock._linearised_investment_constraints